1224-user_administration/1424-administrator-configures-automatic-creator-role.feature

```diff
@@ -0,0 +1,14 @@
+@1424
+Scenario Outline: Administrator configures automatic Creator role
```

Comment:
When I replace the `<Option>` with one of the respective strings the scenario does not make sense. It makes sense when I think that the Options are listed below but `<Option>` is a placeholder. I think this is the reason why the categories are not used and the reason why it is not specified that the categories "employee, member, stuff" are assigned the Creator role.

```suggestion
Given that the User is logged in as Administrator
And they are on the User Administrator Page
And they see a list of checkboxes in the Registrator section
And they see the checkbox with text <Option>
When they select <Option>
And they click the Save button
Then they see a Notifcation that the users with Feide catogory <Category> have got the Creator role
```

Reply:
Not sure if I understand 100%, but I think the Feide category may be too technical for a regular Institution admin, e.g. I'm not sure if it makes sense for them to get a message similar to `Users with Feide category member, employee, faculty have got the Creator role`, since I think they will have knowledge to the `<Option>` part, while `<Category>` is a more techincal thing the user don't need to know. It's possible to select both options (and more if added later), so this will further complicate feedback message.
However, `<Category>` is important for how this is implemented in backend, so I am not sure how this should be represented in the scenario...